src/client/components/styled-components/AddOrganizationError.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+const AddOrganizationError = props => (
+  <svg width="86px" height="86px" viewBox="0 0 86 86" {...props}>
+    <g
+      id="Profiles"
+      stroke="none"
+      strokeWidth="1"
+      fill="none"
+      fillRule="evenodd"
+    >
+      <g
+        id="Add-Organization"
```

Comment:
Not sure what the id here does, but should we pass as an optional prop, and if not, use a UUID? Do we even need it?

Reply:
Documentation says "a unique identifier for distinguishing this element from all other elements in the document" - so it's the id as we understand it. It is also optional - we don't use it anywhere, it is omnipresent and it seems to contain random values. Requirement, however, is that ids are unique, therefore I removed them from all svgs in styled-components.